packages/databricks-vscode/src/run/WorkflowRunner.ts

```diff
@@ -86,9 +88,16 @@ export class WorkflowRunner implements Disposable {
         syncDestination: SyncDestinationMapper;
         token?: CancellationToken;
         connectionManager?: ConnectionManager;
+        wsfsVerifier: WorkspaceFsAccessVerifier;
     }) {
         const panel = await this.getPanelForUri(program);
-
+        if (!wsfsVerifier?.verifyCwdForCluster(cluster)) {
+            panel.setWarning(
+                `For clusters with dbr < 13 (current dbr: ${cluster.dbrVersion.join(
+                    "."
+                )}) the current working directory is set to a temporary folder within driver.`
```

Comment:
Let's be more specific. The working directory is going to be `/databricks/driver`